test/conformance/testdata/conformance.yaml

```diff
@@ -933,6 +933,13 @@
     visible to subsequent reads.
   release: v1.22
   file: test/e2e/apps/statefulset.go
+- testname: Conformance tests minimum number of nodes.
+  codename: '[sig-architecture] Conformance Tests should have at least two untainted
```

Comment:
If this is a requirement of the tests then it should be a requirement of specific tests (and ... it is, see discussion on the issue). I don't think we have any other tests that are solely checking for the cluster to meet the requirements of another test case.